workspaces/cli-server/src/spectacle/index.ts

```diff
@@ -97,9 +132,10 @@ export class LocalCliSpecRepository implements IOpticSpecReadWriteRepository {
 
   async listEvents(): Promise<any[]> {
     if (isEnvTrue(process.env.OPTIC_ASSEMBLED_SPEC_EVENTS)) {
-      const events = OpticEngine.readSpec({
+      const streamArray = StreamArray.withParser();
```

Comment:
I'm not sure how this is intended to be used, because if it's going to use a stream to avoid loading this into memory - all instances will need to be updated to consume a stream